### PR TITLE
Import as csv for pr

### PR DESCRIPTION
**Pull request by FeizNouri**

### CODE REVIEW

File: historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/ImportRequestParser.java [hunk 1]
```diff
@@ -78,21 +94,59 @@ public ResponseAndErrorHolder checkAndBuildValidHistorianImportRequest(JsonArray
...
         public ArrayList<String> errorMessages;
+        public JsonObject numberOfFailedPointsPerMetric;
 
         ResponseAndErrorHolder () {
             this.correctPoints = new JsonArray();
             this.errorMessages = new ArrayList<>();
+            this.numberOfFailedPointsPerMetric = new JsonObject();
+        }
+    }
+    public static class ResponseAndErrorHolderAllFiles {
```
Comment (MiniPlayer) [inline historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/ImportRequestParser.java:137]: why declaring this as a nested static class of ImportRequestParser ? This is not even used in ImprotRequestParser class.

File: historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/ImportRequestParser.java [hunk 2]
```diff
@@ -78,21 +94,59 @@ public ResponseAndErrorHolder checkAndBuildValidHistorianImportRequest(JsonArray
...
+
+        public JsonObject correctPoints;
+        public ArrayList<ArrayList<String>> errorMessages;
+        public JsonArray tooBigFiles;
+        public JsonObject numberOfFailedPointsPerMetric;
```
Comment (MiniPlayer) [inline historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/ImportRequestParser.java:142]: Why using not typed object ? I mean tooBigFIles as JsonArray so we absolutely does not know what it should correspond to... As someone that does not know the code...

Comment (FeizNouri) [reply historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/ImportRequestParser.java:142]: i change it to namesOfTooBigFiles

File: historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/IngestionApiImpl.java [hunk 3]
```diff
@@ -37,9 +30,12 @@ public IngestionApiImpl(HistorianService service) {
     @Override
     public void importJson(RoutingContext context) {
         final ImportRequestParser.ResponseAndErrorHolder responseAndErrorHolder;
+        final ImportRequestParser.ResponseAndErrorHolderAllFiles responseAndErrorHolderAllFiles = new ImportRequestParser.ResponseAndErrorHolderAllFiles();
```
[the remark this answers is not in the document]
Comment (FeizNouri) [reply historian-server/src/main/java/com/hurence/webapiservice/http/api/ingestion/IngestionApiImpl.java:33]: done

File: historian-server/src/main/java/com/hurence/webapiservice/historian/impl/SolrHistorianServiceImpl.java [hunk 4]
```diff
@@ -354,35 +336,62 @@ public HistorianService getAnnotations(JsonObject params, Handler<AsyncResult<Js
     }
 
     @Override
-    public HistorianService addTimeSeries(JsonArray timeseries, Handler<AsyncResult<JsonObject>> resultHandler) {
-
+    public HistorianService addTimeSeries(JsonObject timeseriesObject, Handler<AsyncResult<JsonObject>> resultHandler) {
+        final String chunkOrigin = timeseriesObject.getString(IMPORT_TYPE, "ingestion-json");
+        final JsonObject failedPoints = timeseriesObject.getJsonObject("number of failed points");
```
[the remark this answers is not in the document]
Comment (FeizNouri) [reply historian-server/src/main/java/com/hurence/webapiservice/historian/impl/SolrHistorianServiceImpl.java:341]: done

File: historian-server/src/integration-test/java/com/hurence/webapiservice/historian/HistorianImportJsonVerticleIT.java [hunk 5]
```diff
@@ -62,8 +62,9 @@ static void finish(SolrClient client, Vertx vertx, VertxTestContext context) thr
     void importJsonTimeseries(VertxTestContext testContext) {
         long time1 = 1477895624866L;
         long time2 = 1477916224866L;
-        JsonArray params = new JsonArray().add(new JsonObject().put(NAME, "openSpaceSensors.Temperature000").put(POINTS_REQUEST_FIELD, new JsonArray().add(new JsonArray().add(time1).add(2.0)).add(new JsonArray().add(time2).add(4.0))));
-        historian.rxAddTimeSeries(params)
+        JsonArray params = new JsonArray().add(new JsonArray().add(new JsonObject().put(NAME, "openSpaceSensors.Temperature000").put(POINTS_REQUEST_FIELD, new JsonArray().add(new JsonArray().add(time1).add(2.0)).add(new JsonArray().add(time2).add(4.0)))));
+        JsonObject paramsObject = new JsonObject().put(CORRECT_POINTS, params).put(IMPORT_TYPE, "ingestion-json");
```
[the remark this answers is not in the document]
Comment (FeizNouri) [reply historian-server/src/integration-test/java/com/hurence/webapiservice/historian/HistorianImportJsonVerticleIT.java:66]: done

**Pull request merged**